Provider/BaseVideoProvider.php

```diff
@@ -83,7 +83,10 @@ public function getReferenceFile(MediaInterface $media)
         } else {
             $referenceFile = $this->getFilesystem()->get($key, true);
             $metadata = $this->metadata ? $this->metadata->get($media, $referenceFile->getName()) : array();
-            $referenceFile->setContent($this->browser->get($this->getReferenceImage($media))->getContent(), $metadata);
+
+            $response = $this->client->request('GET', $this->getReferenceImage($media));
```

Comment:
you need the request factory to create a PSR-7 request. the client is no longer a factory for requests with PSR-7.